cmd/version.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+  "fmt"
+
+  "github.com/spf13/cobra"
+)
+
+var (
+      VERSION = "0.0.3"
```

Comment:
Instead of having a hardcoded value here can we fetch the latest tag using either of:
`git describe --tag --abbrev=0`
or
`git tag -l|head -n 1`

Reply:
I thought we wanted to have the hash so we know what commit was used. But I think both ways are good

Reply:
Yes we do but in this format: `0.0.3 (f825de1)`